src/utils/filter_utils.ts

```diff
@@ -1,7 +1,7 @@
 import * as ethUtil from 'ethereumjs-util';
 import * as jsSHA3 from 'js-sha3';
 import * as _ from 'lodash';
-import * as uuid from 'uuid/v4';
+import uuid from 'uuid/v4';
```

Comment:
Would you mind doing the same with lodash and the other packages that are being imported with * accross the project?